### ISSUE TITLE
Pass the country from the value on creation

### ISSUE THREAD

**Issue by loonpwn** [EducationLink/vue-tel-input#51]
When you pass in a value for the input which has the country code prefix, the expected behaviour is that the component will initialise on the country with the prefix. 

```
phone: '+93 34 234 3434'
```

```
 vue-tel-input(v-model="phone")
```

Instead it will set the country based on the users country. In my case, it will always show Australia regardless of the phone prefix. 

The implementation of this is fairly straight forward, the country can be found using the below code and set programatically. For fixing this myself without forking this repo, i've passed the default country property as the parsed country code.

```
import { parsePhoneNumberFromString } from 'libphonenumber-js'
...
this.country = parsePhoneNumberFromString(this.phone).country
...
 vue-tel-input(v-model="phone" :default-country="this.country")
```

**Comment by cresilva** [comment]
You can try this:

Set ref property:

```
<vue-tel-input
              name="mobilePhone"
              :preferredCountries="['br', 'us']"
              placeholder="Ex.: +55 21 9999 99999"
              **ref="telInput"**
            ></vue-tel-input>
```

and 

call this method:

```
    selectCountry: function() {
      this.$refs.telInput.choose(
        this.$refs.telInput.findCountry("ISO_CODE")
      );
    }
```

**Comment by loonpwn** [comment]
This is a bug not an enhance or help wanted. Thanks for your help @cresilva but this is not helpful. I have already posted my work around for the issue.

The point of this issue is that the country should be selected based on the value provided, this is what a user would expect and what a developer would expect to happen.

**Comment by iamstevendao** [comment]
@loonpwn @cresilva Sorry for the delay guys, it should be fixed on c73d0099faabca80f831cc29857ea18a580e2682 (`v2.2.1`)
